src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteWaypointModel.js

```diff
@@ -294,16 +294,12 @@ export default class StandardRouteWaypointModel extends BaseModel {
     }
 
     // TODO: why do we need to clone?
-    // TODO ClonePoisitonFromFix -> clonePositionFromFix
```

Comment:
It could, but a little extra verbosity never hurts.

Reply:
I guess the point was that it sounds like the name of a method that would take an argument: `.clonePositionFromFix(fixName)`, when it actually does not. It just looks up and clones the position of the FixModel for which the StandardRouteWaypointModel is named. As I look again, it seems more okay now. Removed the note.

Reply:
🤔 thats a really good point.  That is certainly a better way to do that because it is a little handily at the moment.  This method is doing too much in that respect, it would be much easier to ask the `FixCollection` for a cloned `PositionModel` instead of _'Give me the fix.  Ok now Mr. Fix, give me a clone of your `PositionModel`.

Maybe add a `TODO` here to update that.  Good callout. 👍